website/docs/r/automation_schedule.html.markdown

```diff
@@ -61,6 +65,26 @@ The following arguments are supported:
 
 * `timezone` - (Optional) The timezone of the start time. Defaults to `UTC`. For possible values see: https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
 
+* `advanced_schedule` - (Optional) Advanced fine-grained schedule frequency configuration. The `advanced_schedule` block supports fields documented below.
+
+---
+
+The `advanced_schedule` block supports:
+
+* `week_days` - (Optional) List of days of the week that the job should execute on.
+
+* `month_days` - (Optional) List of days of the month that the job should execute on. Must be between 1 and 31. 0 for last day of the month.
+
+* `monthly_occurrence` - (Optional) List of occurrences of days within a month. The `monthly_occurrence` block supports fields documented below.
+
+---
+
+The `monthly_occurrence` block supports:
+
+* `day` - (Required) Day of the occurrence. Must be one of Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday.
+
+* `occurrence` - (Required) Occurrence of the week within the month. Must be between 1 and 5.
```

Comment:
*question* tried with the Azure Rest API site, I also see `-1` is available here (e.g. `"Monday" -1` means the last Monday of a month). Can you confirm whether it is supported or not?

Reply:
It's supported. Confirmed and added.